share/completions/go.fish

```diff
@@ -36,18 +45,28 @@ complete -c go -n __fish_use_subcommand -a build -d 'compile packages and depend
 
 # clean
 complete -c go -n __fish_use_subcommand -a clean -d 'remove object files'
-complete -c go -n '__fish_seen_subcommand_from clean' -s i -d "remove the corresponding installed archive or binary (what 'go install' would create)"
+complete -c go -n '__fish_seen_subcommand_from clean' -s i -d "remove the corresponding installed archive or binary"
 complete -c go -n '__fish_seen_subcommand_from clean' -s n -d "print the remove commands it would execute, but not run them"
-complete -c go -n '__fish_seen_subcommand_from clean' -s r -d "clean to be applied recursively to all the dependencies of the packages named by the import paths"
+complete -c go -n '__fish_seen_subcommand_from clean' -s r -d "clean to be applied recursively to all the dependencies"
 complete -c go -n '__fish_seen_subcommand_from clean' -s x -d "clean to print remove commands as it executes them"
+complete -c go -n '__fish_seen_subcommand_from clean' -o cache -d 'clean to remove the entire go build cache'
+complete -c go -n '__fish_seen_subcommand_from clean' -o testcache -d 'clean to expire all test results in the go build cache'
+complete -c go -n '__fish_seen_subcommand_from clean' -o modcache -d 'clean to remove the entire module download cache'
 
 # doc
 complete -c go -n __fish_use_subcommand -a doc -d 'run godoc on package sources'
-complete -c go -n '__fish_seen_subcommand_from doc' -s n -d "prints commands that would be executed"
-complete -c go -n '__fish_seen_subcommand_from doc' -s x -d "prints commands as they are executed"
+complete -c go -n '__fish_seen_subcommand_from doc' -o all -d "Show all the documentation for the package"
```

Comment:
These are all capitalized. Either is fine, but consistency is preferable.

Reply:
Okay, no, these should all be capitalized, but that can be done in a later commit.